### ISSUE TITLE
Whitelist per TFA domain?

### ISSUE THREAD

**Issue by Webunity** [thomseddon/traefik-forward-auth#57]
Precondition: I have one TFA instance deployed, with for example google auth configured.
i want to secure both domain1 & domain2; but domain1 i would only want to allow some@user.com and for domain2 i want to allow both users. Is this currently possible to use docker compose and inject separate whitelists per container?

**Comment by thomseddon** [comment]
No! But it's a good idea, this could be something that we integrate into the rules config.

**Comment by Webunity** [comment]
@thomseddon maybe it would be an idea to have more options configurable per domain (for example clientId/Secret to still allow different login flows but with only one TFA instance). P.s. is the `lifetime` cookie setting also thesame name in v1?

**Comment by thomseddon** [comment]
This has been proposed in #63

**Comment by cstack89** [comment]
This is similar to what I'm looking for, but a little different. I'd like to look at the scope of the token and have configurable rules for that. Like securing /admin to only allow users with an admin scope. 

Also instead of having rules at this level, could they be passed in from traefik on the URL? Like /auth?scope=admin or /auth?username=joe
